.github/workflows/actions.yml

```diff
@@ -18,41 +18,21 @@ jobs:
          - name: "Unit tests"
            test_suite: 'test jacoco:report'
            python: 3.6
-           java: 8
+           java: 11
          - name: "Style check"
            test_suite: 'compile com.mycila:license-maven-plugin:check pmd:pmd pmd:cpd pmd:check pmd:cpd-check javadoc:jar'
            python: 3.6
-           java: 8
-         - name: "OSGi integration"
-           test_suite: 'install -P docker-integration-test,osgi-integration-tests -Dit.cassandra.version=3.0 -DskipUTs'
-           python: 3.6
-           java: 8
-         - name: "Standalone integration 3.0"
-           test_suite: 'verify -P docker-integration-test,standalone-integration-tests -Dit.cassandra.version=3.0 -DskipUTs'
-           python: 3.6
-           java: 8
-         - name: "Standalone integration 3.11"
-           test_suite: 'verify -P docker-integration-test,standalone-integration-tests -Dit.cassandra.version=3.11 -DskipUTs'
-           python: 3.6
-           java: 8
-         - name: "Standalone integration 4.0 (Java 8)"
-           test_suite: 'verify -P docker-integration-test,standalone-integration-tests -Dit.cassandra.version=4.0 -DskipUTs'
+           java: 11
+         - name: "OSGi integration 4.0 (Java 11)"
+           test_suite: 'install -P docker-integration-test,osgi-integration-tests -DskipUTs'
            python: 3.6
-           java: 8
+           java: 11
          - name: "Standalone integration 4.0 (Java 11)"
-           test_suite: 'verify -P docker-integration-test,standalone-integration-tests -Dit.cassandra.version=4.0 -DskipUTs'
+           test_suite: 'verify -P docker-integration-test,standalone-integration-tests -DskipUTs'
            python: 3.6
            java: 11
-         - name: "Python 2.7 integration (Cassandra 3.11, Java 8)"
+         - name: "Python integration (Cassandra 4.0, Java 11)"
            test_suite: 'verify -P docker-integration-test,python-integration-tests -DskipUTs'
-           python: 2.7
-           java: 8
-         - name: "Python 3.6 integration (Cassandra 3.11, Java 8)"
-           test_suite: 'verify -P docker-integration-test,python-integration-tests -DskipUTs'
-           python: 3.6
-           java: 8
-         - name: "Python 3.6 integration (Cassandra 4.0, Java 11)"
-           test_suite: 'verify -P docker-integration-test,python-integration-tests -Dit.cassandra.version=4.0 -DskipUTs'
            python: 3.6
            java: 11
```

Comment:
This is not really necessary to specify anymore on each matrix entry. We can put the java version directly into the step.